web/src/test/manual/build.sh

```diff
@@ -19,22 +19,24 @@ builder_parse "$@"
 DEST="web/build/test-resources"
 
 builder_describe_outputs \
-  build       "/$DEST/sentry-manager.js"
+  build       "${KEYMAN_ROOT}/${DEST}/sentry-manager.js"
```

Comment:
For better or worse, outputs are relative to root of repo.

```suggestion
  build       "/${DEST}/sentry-manager.js"
```

Reply:
Done